### DISCUSSION TITLE
Running LAMMPS with PLUMED

### DISCUSSION THREAD

**Discussion by namtranlab** [deepmodeling/deepmd-kit#1551] [Q&A] [answered]
Dear DeepMD-kit experts,

When installing DeepMD-kit with conda or offline version. 
A plugin version of LAMMPS is also built. 
I would like to run metadynamics using the built-in version of LAMMPS together with PLUMED.
Is there any way we can integrate PLUMED?

Best,

**Comment by njzjz** [comment, marked as answer]
Please refer [LAMMPS doc](https://docs.lammps.org/Build_extras.html#plumed) to integrate LAMMPS with PLUMED. This is not related to DeePMD-kit.
Note: The LAMMPS you installed from conda has already enabled PLUMED.

**Reply by wanghan-iapcm** [reply]
The problem is if a user wants to use both deepmd and plumed in lammps, does he/she have to rebuild lammps? Or is there anyway of using plumed in our released lammps+deepmd binary?

**Reply by njzjz** [reply]
Our released lammps+deepmd binary has been already built with plumed.

**Reply by namtranlab** [reply]
Sorry! I found a similar thread #1277 .

It seems we can directly run deepmd+lammps and plumed without the need to rebuild the package.

Thank you for your help.
Best
